src/screen.rs

```diff
@@ -1,3 +1,4 @@
+#![allow(dead_code)]
```

Comment:
I'd rather keep the warnings on, that we can track down unused functions (example: redraw() can be deleted)